src/tictactoe/tictactoe_game.rs

```diff
@@ -1,6 +1,6 @@
 use crate::game::common::{GameColor, GameMove, GamePlayer, GamePosition, IGame};
 
-pub const BOARD_SIZE: usize = 3;
+pub const BOARD_SIZE: u8 = 3;
```

Comment:
Why do you prefer this? Not that `u8` bothers me, just wondering why make an effort to change it